pkgs/development/python-modules/oterm/default.nix

```
@@ -0,0 +1,42 @@
+{ stdenv, lib, buildPythonPackage, fetchFromGitHub
+, poetry-core, textual, typer, python-dotenv, httpx
+, aiosql, aiosqlite, pyperclip, packaging
+, ruff, black, pytest, pytest-asyncio, textual-dev, aiohttp }:
+
+buildPythonPackage rec {
+  pname = "oterm";
+  version = "0.1.16";
```

```suggestion
  version = "0.1.16";
  pyproject = true;
```
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters